Files/Views/SettingsPages/Preferences.xaml

```diff
@@ -119,6 +119,13 @@
                     Header="Open folders in new tab"
                     HeaderTemplate="{StaticResource CustomHeaderStyle}"
                     IsOn="{Binding OpenFoldersNewTab, Mode=TwoWay}" />
+
+                <ToggleSwitch
+                    x:Name="OpenFoldersNewTabCtrlLeftClick"
+                    x:Uid="SettingsPreferencesOpenFoldersNewTabCtrlLeftClick"
+                    Header="Ctrl + Left mouse click to open folders in new tab"
+                    HeaderTemplate="{StaticResource CustomHeaderStyle}"
+                    IsOn="{Binding OpenFoldersNewTabCtrlLeftClick, Mode=TwoWay}" />
```

Comment:
Can you remove the setting for this since it should be the default behavior?

Reply:
Are you sure about this being default? So you won't be able to Ctrl+Click to select multiple items?

Reply:
Oh, this is for folders? I got mixed up with the sidebar.

Reply:
In that case, it might be best not to add this behavior add all. There is already support for middle clicking on the mouse to open in a new tab.

Reply:
Yeah this PR is for folders but I'm not sure if this was what #4511 was supposted to be about

Reply:
> In that case, it might be best not to add this behavior add all
Exactly

Reply:
Let's close this for now.